config/routes.rb

```diff
@@ -14,37 +14,82 @@
     as: :ontology_iri_versioned,
     constraints: [
       RefLocIdRouterConstraint.new(Ontology, ontology: :id),
-      MIMERouterConstraint.new('text/plain', 'text/html')
-    ]
+    ] do
+      accept 'text/html'
+      accept 'text/plain'
+      accept 'application/json'
+
+      doc title: 'Ontology IRI (loc/id) with version reference',
+          lang: 'de',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.